ADJ-Internship/BusinessService/Dtos/PODisplayDto.cs

```diff
@@ -0,0 +1,50 @@
+﻿using ADJ.BusinessService.Core;
+using ADJ.DataModel.Core;
+using ADJ.DataModel.OrderTrack;
+using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace ADJ.BusinessService.Dtos
+{
+    public class PODisplayDto : EntityDtoBase
```

Comment:
sometimes it is "PO", sometimes it is "Order"
just choose 1 and make it consistent